ocp-templates/mpp/frontend.yaml

```diff
@@ -7,32 +7,33 @@ metadata:
 # Frontend
 # ===============================================
 objects:
-- kind: DeploymentConfig
-  apiVersion: v1
+- kind: Deployment
+  apiVersion: apps/v1
   metadata:
     labels:
-      app: ${APP_NAME}
+      app: ibutsu-frontend
```

Comment:
I would use same app name every where.  If we want separate labeling for frontend and backend we can add that as well.

Reply:
For Deployment objects, the label is used for the match pod policy, so in this case, every pod requires different labels.

I agree though, that for consistency, we can have the same set of labels for the same app, but I don't feel it should be part of this PR and more like part of a followup. I don't want to change labels across all objects right now, but happy to do in a followup